shell/common/asar/archive.cc

```diff
@@ -141,6 +142,9 @@ Archive::~Archive() {
 }
 
 bool Archive::Init() {
+  // Should only be initialized once
+  CHECK(!header_);
```

Comment:
Ah hm, I just realized that `header_ == nullptr` is one of the valid outcomes of calling `Init()`, so we should perhaps use a separate `initialized_` variable to track whether or not this function has been called previously.